### ISSUE TITLE
Regression: cannot edit image content

### ISSUE THREAD

**Issue by gallardo** [alkacon/opencms-core#462]
(Related to #442)

Affected versions: docker container `alkacon/opencms:10.5.0`, (last) current git version (https://github.com/alkacon/opencms-core/commit/99ba95829b0c97407c5cf4a87219e3433c9f1779)
This still works in the docker container `alkacon/opencms:10.0.1`

# Steps to reproduce:

1. Start docker container `alkacon/opencms:10.5.0`
2. Login as admin in `login/index.html`
3. Edit slider images (cs_00007):
![screenshot_2016-12-13_15-07-44](https://cloud.githubusercontent.com/assets/790448/21143209/04ad3d8a-c146-11e6-9cdf-95da8daadbec.png)
4. Open Image Gallery
![screenshot_2016-12-13_15-09-07](https://cloud.githubusercontent.com/assets/790448/21143244/229b147a-c146-11e6-97ae-3dba7524d3f1.png)

# Sample GUI log
N.B.: Server log is identical
```
(TypeError) : a is null

Ticket Id: 1481637760518
com.google.gwt.core.client.JavaScriptException: (TypeError) : a is null
Unknown.Kd(http://localhost:8989/opencms/handleStatic/gwt/opencms/007EBA2E05B7BEE8C7052EBBACE66871.cache.html)
Unknown.Nsf(http://localhost:8989/opencms/handleStatic/gwt/opencms/007EBA2E05B7BEE8C7052EBBACE66871.cache.html)
Unknown.Wdf(http://localhost:8989/opencms/handleStatic/gwt/opencms/007EBA2E05B7BEE8C7052EBBACE66871.cache.html)
Unknown.ldf(http://localhost:8989/opencms/handleStatic/gwt/opencms/007EBA2E05B7BEE8C7052EBBACE66871.cache.html)
Unknown.ndf(http://localhost:8989/opencms/handleStatic/gwt/opencms/007EBA2E05B7BEE8C7052EBBACE66871.cache.html)
Unknown.JZd(http://localhost:8989/opencms/handleStatic/gwt/opencms/007EBA2E05B7BEE8C7052EBBACE66871.cache.html)
Unknown.Wkc(http://localhost:8989/opencms/handleStatic/gwt/opencms/007EBA2E05B7BEE8C7052EBBACE66871.cache.html)
Unknown.ER(http://localhost:8989/opencms/handleStatic/gwt/opencms/007EBA2E05B7BEE8C7052EBBACE66871.cache.html)
Unknown.SR(http://localhost:8989/opencms/handleStatic/gwt/opencms/007EBA2E05B7BEE8C7052EBBACE66871.cache.html)
Unknown.onreadystatechange<(http://localhost:8989/opencms/handleStatic/gwt/opencms/007EBA2E05B7BEE8C7052EBBACE66871.cache.html)
Unknown.yD(http://localhost:8989/opencms/handleStatic/gwt/opencms/007EBA2E05B7BEE8C7052EBBACE66871.cache.html)
Unknown.BD(http://localhost:8989/opencms/handleStatic/gwt/opencms/007EBA2E05B7BEE8C7052EBBACE66871.cache.html)
Unknown.AD/<(http://localhost:8989/opencms/handleStatic/gwt/opencms/007EBA2E05B7BEE8C7052EBBACE66871.cache.html)
Unknown.anonymous(Unknown)
```

**Comment by tHerrmann** [comment]
We fixed this with commit 8606c7e66ef1e19527750b60584ee141c6dcae38.

**Comment by gallardo** [comment]
:cry: You're completely right. My fault. I did test switching only `opencms.jar`, but not `org.opencms.gwt.jar`.
